src/index.js

```diff
@@ -124,71 +124,72 @@ const init = async () => {
   //app.use(asyncMiddleware(setApproverContext));
   app.use(setConfigContext);
 
-
   app.use(function (req, res, next) {
-    let userId = null;
-
-    if (req.user && req.user.sub) {
-      userId = req.user.sub;
+    if (req.originalUrl.indexOf('healthcheck') === -1) {
```

Comment:
checking for undefined 

if (req.originalUrl && req.originalUrl.indexOf('healthcheck') === -1)